src/lwc/rd2ElevateCreditCardForm/rd2ElevateCreditCardForm.js

```diff
@@ -82,7 +82,10 @@ export default class rd2ElevateCreditCardForm extends LightningElement {
     * @description Method sends a message to the visualforce page iframe requesting a token.
     */
     requestToken() {
+        this.clearError();
```

Comment:
When an invalid card is entered,  an error is displayed on the RD form as expected. If the Amount is changed to a negative number but the credit card information is fixed, the error on the Elevate credit card widget should not be displayed.
@rponti-sforg